redi-connect-front/src/components/organisms/Navbar.tsx

```diff
@@ -1,57 +1,47 @@
 import React, { useState } from 'react'
-import { Section, Container, Content } from 'react-bulma-components'
-import { useHistory } from 'react-router-dom'
-import Button from '../atoms/Button'
+import { Section, Container, Element } from 'react-bulma-components'
 import Logo from '../atoms/Logo'
 import burger from '../../assets/images/hamburger.svg'
-import './LoggedOutNavbar.scss'
+import classnames from 'classnames'
+import './Navbar.scss'
 
-const LoggedOutNavbar = () => {
-  const [menuActive, setMenuActive] = useState(false)
+interface Props {
```

Comment:
No need to declare `children` here, instead use `FunctionComponent`, just as done in another file changed in this commit: `/src/components/templates/Landing.tsx`

For exhaustive information: https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#function-components

Reply:
sure, I also aready have a few sections where I will need more support considering TS from you. but for now lets keep moving